_config.yml

```diff
@@ -10,13 +10,13 @@ sass:
 
 # Resume settings
 resume_avatar:                  "true"
-resume_name:                    "Skatetocat"
```

Comment:
resume_title:                   "Professional Skater"
resume_contact_email:           "skatetocat@github.com"
resume_name:                    "GitHub Teacher"
resume_title:                   "Professional Trainer"
resume_contact_email:           "services@github.com"
resume_contact_telephone:       "1-(877)-448-4820"
resume_contact_address:         "San Francisco, California"
resume_header_contact_info: "San Francisco, California | 1-(877)-448-4820 | skatetocat@github.com"
resume_header_intro: "<p>Professional Skater, 'nuff said.</p>"
resume_header_contact_info: "San Francisco, California | 1-(877)-448-4820 | services@github.com"
resume_header_intro: "<p>Charting the knowledge of the Internet, just like Galileo charted the stars.</p>"